src/actions/user.actions.ts

```diff
@@ -34,6 +34,15 @@ export namespace User {
         userData: ApiUser
       }) {}
     }
+
+    export class SetRegisterLoading {
+      static type = 'USER_SET_REGISTER_LOADING';
+      type = SetRegisterLoading.type;
+      constructor(public payload: {
+        isRegisterLoading: boolean
+      }) {}
+    }
+
```

Comment:
Not sure if this info is worth using redux for or not. If not, I can switch it over to not using redux.

Reply:
Yes, this seems like it should be a state variable in the registration page.

Reply:
Done. This now uses a state variable.